core/src/main/scala/precog/AutoBump.scala

```diff
@@ -80,13 +80,14 @@ object AutoBump {
     val order: Map[ChangeLabel, Int] = values.zipWithIndex.toMap
     implicit val ordering: Order[ChangeLabel] = Order.by(order)
     val fromString: Map[String, ChangeLabel] = values.map(change => change.label -> change).toMap
-    val labelPattern: Regex = values.mkString("|").r
+    val labelPattern: Regex = values.map(_.label).mkString("|").r
 
     def apply(label: String): Option[ChangeLabel] = {
       fromString.get(label)
     }
 
-    def unapply(arg: String): Option[ChangeLabel] = labelPattern.findFirstIn(arg).flatMap(apply)
+    def unapply(arg: String): Option[ChangeLabel] =
+      labelPattern.findFirstIn(arg).flatMap(apply)
```

Comment:
What's the right margin you are using?

Reply:
It's about 100. I basically forcibly wrap whenever side-by-side panes in an editor with minimal chrome, 12 point font, and 2x scaling on a MacBook Pro begins to clip.

![Screen Shot 2020-04-14 at 1 45 00 PM](https://user-images.githubusercontent.com/752/79267290-2cac5480-7e56-11ea-8e86-cbcb7a2b3b3e.png)